pages/clubs-list.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require('react');
```

Comment:
Added basic structure to this new page. Real content were added in succeeding PRs